### DISCUSSION TITLE
Customisation of Author social links

### DISCUSSION THREAD

**Discussion by Mecanik** [razonyang/hugo-theme-bootstrap#541] [Q&A] [answered]
Hi,

For me, it makes no sense to have the same social links you can have in the footer. As such, I`m trying to alter my author info to include only Github, Stackoverflow, etc - all OK.

In the footer I am leaving only RSS (which makes sense) and no other social links. However, I am unable to remove the RSS from the author panel; it makes no sense to be there anyway.

On another note the "Contact Us" refers to multiple people, it should be just "Contact".

So my questions are:

- How to remove the RSS from the author panel?
- How to edit the "Contact Us" text to "Contact" (single individual)

Many thanks

**Comment by razonyang** [comment, marked as answer]
There is no parameter for removing the RSS link from social links now, I'll create one for this in next release.

The `Contact US` is the `title` of page `/content/contact/index.md`.

**Reply by Mecanik** [reply]
Thanks, removing the RSS from author would be essential. Regarding the title, it's used as the actual link as well?

**Reply by chrisrummel** [reply]
Please excuse my intrusion, I'd like to share a thought.

While I don't mind havin the RSS-feed in the author box, it would be interesting for sites with multiple authors. In that case the author box should contain a link to an RSS-feed with all posts of this author, and at the bottom of the page should be a link to the sites RSS-feed.
But being able to remove it by uncommenting an option in the config\_default\author.toml is the flexible solution.

**Reply by Mecanik** [reply]
Yep, that would make sense. Separate RSS would be the best option, however in my case I would just remove it. I doubt someone will use this for multiple authors since it's a static website, without a user system, etc.

**Comment by razonyang** [comment]
@Mecanik @chrisrummel  Just created a PR #544 for this, any thoughts are welcome.

```
rss = "" # Disable RSS link
```
